styles.css

```diff
@@ -0,0 +1,435 @@
+  font-weight: 300;
+}
+
+hr {
+  border: 0;
+  border-top: 1px solid #e2e2e2;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover,
+button:hover {
+  cursor: pointer;
+  opacity: 0.8;
+  transition-duration: 0.2s;
+}
+
+.square-btn {
+  background-color: #567b95;
+  color: white;
+  font-weight: bold;
+  border: none;
+  border-radius: 3px;
+}
+
+/* NAVBAR PROPERTIES ========================================================= */
+
+.navbar {
+  background-color: white;
+  position: fixed;
+  top: 0;
+  border-bottom: 1px solid #e2e2e2;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-size: 14px;
+  font-weight: bold;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  z-index: 1;
+}
+
+/* Navbar Sections (Left, Middle, Right part) */
+
+.nav-div {
+  margin: 4px 8%;
+}
+
+/* Navbar Menu (Left part) */
+
+.nav-menu {
+  display: flex;
+  align-items: center;
+}
+
+.nav-menu i {
+  margin-right: 18px;
+}
+
+.nav-menu p {
+  display: inline;
+  margin: 0;
+}
+
+/* Navbar Logo (Middle part) */
+
+.nav-logo {
+  margin-top: 8px;
+}
+
+/* Navbar Buttons (Right Part) */
+
+.nav-btns button {
+  font-size: 11px;
+  padding: 8px 16px;
+}
+
+/* EDITORS PICK PROPERTIES =================================================== */
+
+.editors-pick {
+  width: 240px;
+  float: right;
+  margin-right: 50px;
+  top: 20px;
+}
+
+.editors-pick h6 {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-weight: bold;
+  margin: 0;
+}
+
+.editors-pick img {
+  float: left;
+  margin-right: 10px;
+}
+
+.editors-pick p {
+  font-size: 14px;
+  line-height: 120%;
+  padding-bottom: 10px;
+}
+
+.editors-pick a {
+  color: #333;
+}
+
+/* ARTICLE PROPERTIES ======================================================== */
+
+/* Article Title */
+
+.article h1 {
+  line-height: 120%;
+  font-style: italic;
+  font-size: 36px;
+  color: black;
+}
+
+.sharing-btn {
+  border: 0;
+  border-radius: 50%;
+  background-color: #ececec;
+  padding: 8px;
+  scale: 0.7;
+  height: 40px;
+}
+
+/* Article Text */
+
+.article {
+  color: #333;
+  line-height: 130%;
+  width: 30%;
+  margin: 0 auto;
+}
+
+/* IMAGE FROM ARTICLE PROPERTIES ============================================= */
+
+.article-img {
+  margin: 0 auto;
+}
+
+.article-main-img {
+  width: 63%;
+}
+
+.article-explanation-img {
+  width: 940px;
+  position: relative;
+}
+
+.article-explanation-img img {
+  z-index: -1;
+}
+
+.row-text-box {
+  position: absolute;
+  display: flex;
+}
+
+.top-4 {
+  bottom: 370px;
+}
+
+.bottom-4 {
+  bottom: 80px;
+}
+
+.single-text-box {
+  width: 220px;
+  margin-right: 20px;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-size: 12px;
+}
+
+.article-final-img {
+  width: 40%;
+}
+
+.article-img h4,
+.article-img h6 {
+  margin: 0;
+}
+
+/* Captions */
+
+.caption {
+  font-size: 17px;
+  color: #666;
+  line-height: 110%;
+  margin: 0;
```

Comment:
Always try to avoid the repetition of the same rule(s) as much as you can. You should please DRY your code in the above blocks. 
![image](https://user-images.githubusercontent.com/65501535/108138438-09ce7a80-70cf-11eb-8ad3-05a944bfd24f.png)

`margin: o` has a number of occurences